index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="assets/styles/style.css">
+    <link href='https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css' rel='stylesheet'>
+</head>
+
+<body>
+    <section class="showcase">
+        <header class="main-header">
+            <nav class="">
+                <a class="logo" href="/">
+                    <h1>Current Logo</h1>
+                </a>
+                <div class="">
+                    <i class='bx bx-menu'></i>
+                </div>
+            </nav>
+        </header>
```

Comment:
[OPTIONAL] You have used semantic HTML very well 💯 👍. To improve upon the semantics of your page, however, I suggest that you take the header tag out of the section tag and put it above the section tag. This will present a form of hierarchy to your page and improve upon the semantics. You can check [here](https://www.w3schools.com/html/html5_semantic_elements.asp) for more information about HTML semantic tags. Also, you can visualize the semantic hierarchy in HTML documents in the screenshot below: 
![image](https://user-images.githubusercontent.com/107795045/234979277-213ee0b7-67bf-4e69-bb24-ffbee4c123cc.png)